testkit/javadsl/src/main/scala/com/lightbend/lagom/javadsl/testkit/ServiceTest.scala

```diff
@@ -245,41 +301,65 @@ object ServiceTest {
     val now = DateTimeFormatter.ofPattern("yyMMddHHmmssSSS").format(LocalDateTime.now())
     val testName = s"ServiceTest_$now"
 
-    val b1 = new GuiceApplicationBuilder()
+    val initialBuilder = new GuiceApplicationBuilder()
       .bindings(sBind[TestServiceLocatorPort].to(testServiceLocatorPort))
       .bindings(sBind[ServiceLocator].to(classOf[TestServiceLocator]))
       .bindings(sBind[TopicFactory].to(classOf[TestTopicFactory]))
       .configure("play.akka.actor-system", testName)
 
     val log = Logger(getClass)
 
-    val b3 =
+    val finalBuilder =
```

Comment:
The following code was very cluttered and poorly formatted. Did some cosmetics rearrangement for make it more readable.

Reply:
Fantastic!